dds_cli/__main__.py

```diff
@@ -1632,3 +1633,46 @@ def list_units(click_ctx):
     ) as err:
         LOG.error(err)
         sys.exit(1)
+
+
+####################################################################################################
+####################################################################################################
+## MOTD #################################################################################### MOTD ##
+####################################################################################################
+####################################################################################################
+
+
+@dds_main.group(name="motd", no_args_is_help=True)
+@click.pass_obj
+def motd_group_command(_):
+    """Group command for managing motd messages.
+
+    Limited to Super Admins.
+    """
+
+
+# ************************************************************************************************ #
+# MOTD COMMANDS ******************************************************************** MOTD COMMANDS #
+# ************************************************************************************************ #
+
+# -- dds motd -- #
+@motd_group_command.command(name="add", no_args_is_help=False)
+@click.pass_obj
+def set_motd(click_ctx):
+    """Add a new MOTD."""
+    try:
+        with dds_cli.motd_manager.MotdManager(
+            no_prompt=click_ctx.get("NO_PROMPT", False),
+            token_path=click_ctx.get("TOKEN_PATH"),
+        ) as setter:
+            message = click.prompt("Enter a new MOTD message", type=str)
+            setter.add_new_motd(message)
```

Comment:
```suggestion
            setter.add_new_motd(message)
```

Reply:
Ok, let's leave it like this now. In the long run, IMO, it will bring better user experience if the commands/subcommands are verified directly. A general solution for all available CLI commands would be better than doing it for certain commands.